### DISCUSSION TITLE
Can bubble events be prevented

### DISCUSSION THREAD

**Discussion by xiaoxiji1234** [lowdefy/lowdefy#1153] [Questions and Help]
Hi, I made a page and wrapped a`Button` with `Box`. Both `Box` and `Button` have corresponding click events. Can clicking `Button` not trigger the click event of `Box`

**Comment by Gervwyk** [comment]
Hi @xiaoxiji1234 unfortunately currently not. 
However, I think we should track this as a use case to unpack what the expected / standard behavior should be since both behaviors can be useful, and I'm not sure this should be a feature.
I'll convert it into an issue for now.
